### ISSUE TITLE
[Menu item] Standards and best practice, create submenu items with content from the handbook

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#60]
Create submenu items with starter content for the main menu item "Standards and best practice".

Some content can be taken from the current [Accessibility Handbook](https://make.wordpress.org/accessibility/handbook/) 

## Submenus

1. WCAG and compiance
2. Difference between WCAG and best practice	
3. Content and images
4. Design and user experience
5. Frontend code
6. Forms
7. WordPress Core
8. Themes
9. Plugins

### WCAG and compliance
It would be nice to have one page about WCAG summerizing WCAG general, WCAG for WordPress, a11y statement for WP itself and ATAG, with links to the info on wp.org
One page to refer to when you talk about WCAG

Content can be taken from:
- [What are the guidelines?](https://make.wordpress.org/accessibility/handbook/test-for-web-accessibility/#what-are-the-guidelines)
- [Core Accessibility Standards](https://make.wordpress.org/accessibility/handbook/markup/core-accessibility-standards/)
- [Accessibility Coding Standards
](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/accessibility/)
### Difference between WCAG and best practice	
- [ WordPress Accessibility Statement and ATAG](https://wordpress.org/about/accessibility/)

### Content and images
- Alt texts for images
- Headings
- Links
- Tables (specific, now to add tables in content)
- Readability
- Digital documents

Take content from:
- [Good link texts](https://make.wordpress.org/accessibility/handbook/best-practices/content/good-link-texts/)
- [Link destinations](https://make.wordpress.org/accessibility/handbook/best-practices/content/link-destinations/)
- [Readability](https://make.wordpress.org/accessibility/handbook/best-practices/content/readability/)
- [Headings in the content](https://make.wordpress.org/accessibility/handbook/best-practices/content/using-headings-in-content/)
- [Alternative text for images](https://make.wordpress.org/accessibility/handbook/best-practices/content/alternative-text-for-images/)
- [Tables](https://make.wordpress.org/accessibility/handbook/markup/tables/)
- [Accessible PDFs](https://make.wordpress.org/accessibility/handbook/best-practices/content/accessible-pdfs/)

### Design and user experience
- Fonts
- Use of color
- Color contrast
- Meaningful (new content)
- Patterns to avoid

Take content from:
- [Design best practices](https://make.wordpress.org/accessibility/handbook/design/)
- [Fonts](https://make.wordpress.org/accessibility/handbook/best-practices/markup/font-sizes-and-resize-text/)
- [Use of color](https://make.wordpress.org/accessibility/handbook/design/use-of-color/
- Patterns to avoid: [Infinite scroll](https://make.wordpress.org/accessibility/handbook/markup/design-patterns/infinite-scroll/)

### Frontend code

- Semantic HTML
- Skip links
- CSS class screen-reader-text
- Skip links
- Heading structure
- Landmarks
- Tables
- ARIA live in WordPress (wp.a11y.speak)
- Dialog modal
- Post excerpts for an archive template

Review and take content from:
- [Design Patterns](https://make.wordpress.org/accessibility/handbook/markup/design-patterns/)
- [Post excerpts for an archive template](https://make.wordpress.org/accessibility/handbook/markup/design-patterns/post-excerpts-for-an-archive-template/)
- [Dialog modal](https://make.wordpress.org/accessibility/handbook/markup/design-patterns/dialog-modal/)
- [Semantic HTML](https://make.wordpress.org/accessibility/handbook/markup/semantic-html/)
[The CSS class screen-reader-text](https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/)
- [ARIA Landmarks](https://make.wordpress.org/accessibility/handbook/markup/aria-landmarks/)
- [Skip links](https://make.wordpress.org/accessibility/handbook/markup/skip-links/)
- [Heading structure](https://make.wordpress.org/accessibility/handbook/markup/heading-structure-in-theme-development/)
- [Accessible Forms](https://make.wordpress.org/accessibility/handbook/markup/web-forms/)
- [Tables](https://make.wordpress.org/accessibility/handbook/markup/tables/)
- [Using ARIA Live in WordPress](https://make.wordpress.org/accessibility/handbook/markup/wp-a11y-speak/)

### Forms

Take content from [Web forms](https://make.wordpress.org/accessibility/handbook/best-practices/markup/web-forms/)

### WordPress Core

Take content from:
[Core Accessibility Standards](https://make.wordpress.org/accessibility/handbook/markup/core-accessibility-standards/)

### Themes

- Theme Accessibility
- Accessibility ready guidelines

For the Accessibility ready guidelines, point to https://make.wordpress.org/themes/handbook/review/accessibility/.

Take content from:
[Theme Accessibility](https://make.wordpress.org/accessibility/handbook/which-themes-can-you-use/).

### Plugins
- Plugin Accessibility
Take content from: [Plugin Accessibility](https://make.wordpress.org/accessibility/handbook/plugin-accessibility/) . Review and check links: what should be transfered to the code content and what should stay here. This content is a bit of everything, linkto to the other documentation would be more useful.
